apps/payment/models.py

```diff
@@ -553,3 +580,106 @@ def _send_receipt(self, payment_date, subject, description, payment_id, items, t
 
     class Meta:
         default_permissions = ('add', 'change', 'delete')
+
+
+class FikenSale(models.Model):
+    stripe_key = models.CharField(
+        'Stripe key',
+        max_length=20,
+        choices=StripeKey.ALL_CHOICES,
+        default=StripeKey.ARRKOM,
```

Comment:
Is it necessary to have a default key? Should it not always be selected? Minor nitpick I guess :p

Reply:
I chose it as a default because other places used Arrkom as the default. I can agree on that it can be better to remove it :)